io.openems.edge.io.shelly/src/io/openems/edge/io/shelly/shelly25/IoShelly25Impl.java

```diff
@@ -75,20 +92,19 @@ public BooleanWriteChannel[] digitalOutputChannels() {
 	public String debugLog() {
 		var b = new StringBuilder();
 		var i = 1;
-		for (WriteChannel<Boolean> channel : this.digitalOutputChannels) {
+		for (BooleanWriteChannel channel : this.digitalOutputChannels) {
 			String valueText;
 			var valueOpt = channel.value().asOptional();
 			if (valueOpt.isPresent()) {
-				valueText = valueOpt.get() ? "x" : "-";
+				valueText = valueOpt.get() ? "ON" : "OFF";
```

Comment:
You did not mention this change in the PR description. We use the notion of `x`, `-` and `?` in many places in OpenEMS. Why change it here?